misc/experimental/rust/celix_bindings/src/celix_bindings.h

```diff
@@ -30,3 +30,6 @@
 #include "celix_framework.h"
 #include "celix_framework_factory.h"
 #include "celix_framework_utils.h"
+
+#include "celix_shell_command.h"
+#include "celix_log_service.h"
```

Comment:
Now all components (including `framework` and `utils`) are optional, these headers may be missing.

For Conan, intra-package and inter-package dependency deduction work like charm. But we need to think about how to make Cargo and Conan work together.